I-TECH/openmrs-module-kenyaemr
# Updated KEMRActivator to point to latest HTMLForms package 34

Updated HTML Forms have the following changes -

Impacted Forms -
Family History
Clinical Encounter - HIV Addendum
Lab Results
Other Medications
Obstetric History

Issue -
All these forms have a table with rows in it which can be added/removed. The current implementation only displays the first row on load and the remaining data is hidden. It gets displayed only on the Add button click.

Resolution -
Remove the "display:none" CSS style as well as the Add/Remove buttons from the HTML forms. All rows are displayed whether it has data or not.

Test case:
Repeat for every form in the list above
1. Open the form and enter one row of data and save.
2. Open the form and enter two more rows of data and save.
3. On reopening the form, you can see all 3 rows of data entered in the prior steps.
